priv/test_files/test_app/sys.config

```
@@ -0,0 +1,6 @@
+[
+    {test_app, [
+        {environment, test},
+        {a_key, ok}
+    ]}
+].
```

Add this line here, just for kicks…

```suggestion
    ]},
    "path/to/another/sys.config"
].
```